src/Paramore.Brighter.MessageScheduler.Quartz/QuartzBrighterJob.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text.Json;
+using Paramore.Brighter.Scheduler.Events;
+using Quartz;
+
+namespace Paramore.Brighter.MessageScheduler.Quartz;
+
+/// <summary>
+/// The Quartz Message scheduler Job
+/// </summary>
+/// <param name="processor"></param>
+public class QuartzBrighterJob(IAmACommandProcessor processor) : IJob
+{
+    public async Task Execute(IJobExecutionContext context)
+    {
+        if (!context.JobDetail.JobDataMap.TryGetString("message", out var obj))
+        {
+            return;
+        }
+
+        if (!context.JobDetail.JobDataMap.TryGetBooleanValue("async", out var async))
+        {
+            return;
+        }
+
+        var id = context.JobDetail.Key.Name;
+        var message = JsonSerializer.Deserialize<Message>(obj!, JsonSerialisationOptions.Options)!;
+        if (async)
+        {
+            await processor.PostAsync(new FireSchedulerMessage { Id = id, Message = message });
```

Comment:
This works if our intent is always to send a message, but on a Send or Publish intent is actually just to delay Send or Publish.